API.md

```diff
@@ -0,0 +1,97 @@
+# Basic API Guide
+
+## Login
+
+* Endpoint: `/login`
+* Verb: `POST`
+* Requires existing session: `false`
+* GET params: 
```

Comment:
nit: Can we remove the blank `GET params` here and below?

Reply:
Done.